tests/test_meta_cat.py

```diff
@@ -44,6 +45,33 @@ def test_save_load(self):
 
         self.assertEqual(f1, n_f1)
 
+    def test_predict_spangroup(self):
+        Span.set_extension('id', default=0, force=True)
+        Span.set_extension('meta_anns', default=None, force=True)
+
+
+        json_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'resources', 'mct_export_for_meta_cat_test.json')
+        self.meta_cat.train(json_path, save_dir_path=self.tmp_dir)
+        self.meta_cat.save(self.tmp_dir)
+        n_meta_cat = MetaCAT.load(self.tmp_dir)
+        assert n_meta_cat.config.general.span_group is None 
+
+        spangroup_name = 'predict_spangroup'
+        n_meta_cat.config.general.span_group = spangroup_name
+        nlp = spacy.blank("en")
```

Comment:
I think it would make sense to separate the generation of this document (i.e this and the following 4 lines) to a separate method. I.e a `_prepare_doc_w_spangroup` method.